scripts/users-permissions.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+USER="admin:pass"
+HEADER="Content-Type: application/json"
+
+# Alias for curl 
+curl="./bin/curl -u ${USER} "$@""
+
+# Create "Teacher" team
+${curl} -d "name=Teacher" grafana:3000/api/teams
+ 
+# Create "instructor" and "staff" user
+${curl} -d "email=staff@localhost&login=staff&password=funfunfun" grafana:3000/api/admin/users
+${curl} -d "email=instructor@localhost&login=instructor&password=funfunfun" grafana:3000/api/admin/users
+
+# Return id of the "Teacher" team
+TEACHER_TEAM_ID=`${curl} 'http://grafana:3000/api/teams/search?name=Teacher' | jq -r '.teams[] | select(.name=="Teacher") | .id'`
+
+# Return id of the "videos" folder FIXME: to be updated to "teachers" folder
+VIDEO_FOLDER_UID=`${curl} "http://grafana:3000/api/folders" | jq -r '.[] | select(.title=="videos") | .uid'`
+
+# Get permission on "videos" folder for the "Teacher team"
+DATA="{\"items\":[{\"role\":\"Viewer\",\"permission\":1},{\"teamId\":$TEACHER_TEAM_ID,\"permission\":1}]}"
+${curl} -H ${HEADER} -d ${DATA} "http://grafana:3000/api/folders/$VIDEO_FOLDER_UID/permissions"
+
+# Return ids of the "instructor" and "staff" users
+INSTRUCTOR_USER_ID=`${curl} 'http://grafana:3000/api/users/lookup?loginOrEmail=instructor@localhost' | jq '.id'`
+STAFF_USER_ID=`${curl} 'http://grafana:3000/api/users/lookup?loginOrEmail=staff@localhost' | jq '.id'`
+
+# Add "instructor" and "staff" users to the "Teacher" team
+${curl} -H ${HEADER} -d "{\"userId\": $INSTRUCTOR_USER_ID}" grafana:3000/api/teams/$TEACHER_TEAM_ID/members
+${curl} -H ${HEADER} -d "{\"userId\": $STAFF_USER_ID}" grafana:3000/api/teams/$TEACHER_TEAM_ID/members
```

Comment:
I would recommend running shellcheck on this script. You will learn a lot using it! :muscle: